src/sonic-config-engine/portconfig.py

```diff
@@ -46,11 +59,16 @@ def readJson(filename):
         print("error occurred while parsing json: {}".format(sys.exc_info()[1]))
         return None
 
-def db_connect_configdb():
+def db_connect_configdb(namespace=None):
     """
     Connect to configdb
     """
-    config_db = ConfigDBConnector()
+    try:
+        if namespace is not None:
+            swsscommon.SonicDBConfig.load_sonic_global_db_config(namespace=namespace)
+        config_db = swsscommon.ConfigDBConnector(use_unix_socket_path=True, namespace=namespace)
+    except Exception as e:
```

Comment:
>Exception [](http://example.com/codeflow?start=11&length=9)

What is the type of exception you are expecting? The general exception type will hide bugs and should be limited.

Reply:
Was added to handle exception if thrown by load_sonic_global_db_config().

Reply:
looks like load_sonic_global_db_config() calls SonicDBConfig::initializeGlobalConfig() which already handled exceptions, and it should be good enough?

Reply:
I have created a PR : https://github.com/Azure/sonic-buildimage/pull/10581 to remove this, I am working on unit-test failure, will merge that PR in master branch first and then take it in to 202012

Reply:
@qiluo-msft https://github.com/Azure/sonic-buildimage/pull/10581 is merged in master branch to fix this comment. Can we merge this PR to 202012 branch and cherry-pick the other PR to 202012.

Reply:
https://github.com/Azure/sonic-buildimage/pull/10960 - PR raised to fix this comment.